debian version patch
Index: systemtap/Makefile.am
===================================================================
--- systemtap.orig/Makefile.am	2010-08-06 12:49:40.790483273 +0530
+++ systemtap/Makefile.am	2010-08-06 13:47:58.770483274 +0530
@@ -76,7 +76,7 @@
 			cp "$(srcdir)/git_version.h" "git_version.h"; \
 		fi; \
 	fi
-	$(GIT_VERSION_CMD) -k -s $(top_srcdir) -o git_version.h
+	sh $(top_srcdir)/debian/debversion.sh $(top_srcdir)/debian/changelog > git_version.h
 	@if test -s "$(srcdir)/git_version.h"; then \
 		if cmp "$(srcdir)/git_version.h" "git_version.h"; then :; \
 		else \
@@ -87,7 +87,7 @@
 	fi
 
 git_version.h:
-	$(srcdir)/git_version.sh -k --srcdir $(srcdir) -o git_version.h
+	sh $(top_srcdir)/debian/debversion.sh $(top_srcdir)/debian/changelog > git_version.h
 
 cscope:
 	cd $(srcdir) && \
