commit d922b6beafda87737d3dd7d0399f7930d5f6fa66
Author: Eugeniy Meshcheryakov <eugen@debian.org>
Date:   Thu Jan 21 11:22:46 2010 +0100

    Add shebangs

diff --git a/testsuite/systemtap.examples/memory/overcommit.stp b/testsuite/systemtap.examples/memory/overcommit.stp
index a080010..60c48cf 100644
--- a/testsuite/systemtap.examples/memory/overcommit.stp
+++ b/testsuite/systemtap.examples/memory/overcommit.stp
@@ -1,3 +1,5 @@
+#! /usr/bin/env stap
+
 probe kernel.function("__vm_enough_memory").return {
   if ($return != 0) {
       printf("overcommit blocked malloc, process %s (%d), %d pages\n",
diff --git a/testsuite/systemtap.examples/memory/vm.tracepoints.stp b/testsuite/systemtap.examples/memory/vm.tracepoints.stp
index 07cee6f..61335d3 100644
--- a/testsuite/systemtap.examples/memory/vm.tracepoints.stp
+++ b/testsuite/systemtap.examples/memory/vm.tracepoints.stp
@@ -1,3 +1,5 @@
+#! /usr/bin/env stap
+
 global slabs
 
 probe vm.kmem_cache_alloc {
