elfutils build depends patch
Index: systemtap/configure.ac
===================================================================
--- systemtap.orig/configure.ac	2010-08-06 13:42:55.000000000 +0530
+++ systemtap/configure.ac	2010-08-06 13:43:02.010483274 +0530
@@ -415,11 +415,11 @@
     save_LIBS="$LIBS"
     AC_CHECK_LIB(dw, dwfl_module_getsym,[],[
       AC_MSG_ERROR([missing elfutils development headers/libraries (install elfutils-devel, libebl-dev, libdw-dev and/or libebl-devel)])],
-      [-Wl,--start-group -ldw -lebl -Wl,--end-group -lelf])
+      [-ldw -lelf])
     AC_CHECK_LIB(dw, dwarf_getelf,[],[
       AC_MSG_ERROR([elfutils, libdw too old, need 0.126+])],
-      [-Wl,--start-group -ldw -lebl -Wl,--end-group -lelf])
-    stap_LIBS="$stap_LIBS -Wl,--start-group -ldw -lebl -Wl,--end-group -lelf"
+      [-ldw -lelf])
+    stap_LIBS="-ldw -lelf"
     LIBS="$save_LIBS"
   else
     # We built our own and stap_LDFLAGS points at the install.
Index: systemtap/configure
===================================================================
--- systemtap.orig/configure	2010-08-06 12:49:40.000000000 +0530
+++ systemtap/configure	2010-08-06 13:46:10.022483274 +0530
@@ -7214,7 +7214,7 @@
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-ldw -Wl,--start-group -ldw -lebl -Wl,--end-group -lelf $LIBS"
+LIBS="-ldw -Wl,--start-group -ldw -Wl,--end-group -lelf $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -7262,7 +7262,7 @@
   $as_echo_n "(cached) " >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-ldw -Wl,--start-group -ldw -lebl -Wl,--end-group -lelf $LIBS"
+LIBS="-ldw -Wl,--start-group -ldw -Wl,--end-group -lelf $LIBS"
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 
@@ -7304,11 +7304,11 @@
       as_fn_error "elfutils, libdw too old, need 0.126+" "$LINENO" 5
 fi
 
-    stap_LIBS="$stap_LIBS -Wl,--start-group -ldw -lebl -Wl,--end-group -lelf"
+    stap_LIBS="$stap_LIBS -Wl,--start-group -ldw -Wl,--end-group -lelf"
     LIBS="$save_LIBS"
   else
     # We built our own and stap_LDFLAGS points at the install.
-    stap_LIBS="$stap_LIBS -Wl,--start-group -ldw -lebl -Wl,--end-group -lelf"
+    stap_LIBS="$stap_LIBS -Wl,--start-group -ldw -Wl,--end-group -lelf"
   fi
 fi
 
